src/processing/resources/page.js

```diff
@@ -2,18 +2,19 @@ import DomProcessor from '../dom';
 import DomAdapter from '../dom/server-dom-adapter';
 import ResourceProcessorBase from './resource-processor-base';
 import whacko from 'whacko';
+import dedent from 'dedent';
 import scriptProcessor from '../script';
 import * as Const from '../../const';
 import * as contentUtils from '../../utils/content';
 
 const BODY_CREATED_EVENT_SCRIPT =
-          `<script type="text/javascript" class="${ Const.SHADOW_UI_SCRIPT_CLASSNAME }">
-               if (window.Hammerhead)
-                   window.Hammerhead.sandbox.node.raiseBodyCreatedEvent();
+          dedent`<script type="text/javascript" class="${ Const.SHADOW_UI_SCRIPT_CLASSNAME }">
+                     if (window.Hammerhead)
+                         window.Hammerhead.sandbox.node.raiseBodyCreatedEvent();
 
-               var script = document.currentScript || document.scripts[document.scripts.length - 1];
-               script.parentNode.removeChild(script);
-          </script>`;
+                     var script = document.currentScript || document.scripts[document.scripts.length - 1];
+                     script.parentNode.removeChild(script);
+                 </script>`;
```

Comment:
It will look much better if it will be reformatted as:
```js
const BODY_CREATED_EVENT_SCRIPT =dedent`
	<script type="text/javascript" class="${ Const.SHADOW_UI_SCRIPT_CLASSNAME }">
		if (window.Hammerhead)
			window.Hammerhead.sandbox.node.raiseBodyCreatedEvent();
            
		var script = document.currentScript || document.scripts[document.scripts.length - 1];
		script.parentNode.removeChild(script);
	</script>
`;
```